### DISCUSSION TITLE
How to show online for Location / Group?

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1393] [Q&A] [answered]
In another thread, I saw a screenshot of an interface which showed "online" for virtual meetings under the Location / Group column.  Ours just shows a blank.

Since most of our meetings are virtual, this column is currently showing mostly empty.

Here is that screenshot I saw. Is there a way to fill in that column like this?

![Region column trouble](https://github.com/code4recovery/12-step-meeting-list/assets/159723102/ae9c0202-2998-45a3-8098-866d3adf8fac)

**Comment by SIA-WEBSUP** [comment, marked as answer]
The screenshot above is using the Legacy UI. 
You are probably using the new TSML UI instead.

If you were using the Legacy UI, you would see Online in green in the Location/Group column and the Address column would be blank:
    ![image](https://github.com/code4recovery/12-step-meeting-list/assets/63924390/2973048f-2f16-410f-9913-8bb58392643f)


In TSML UI the Location / Group column is blank for Online meetings... but you do have a nice 'Zoom Icon' in the Address column:
    ![image](https://github.com/code4recovery/12-step-meeting-list/assets/63924390/4be5dc0d-0bf5-4313-8999-3bdd12d64404)

**Comment by willinrecovery** [comment]
Ah, thank you for the clarification. Either way, we get a mostly blank column since most of our meetings are online. 

I think I prefer the Zoom button right there, but I will bring this to the website team to see what the group prefers.

Thank you again for your service!
